wiki/whitelist-process.md

```diff
@@ -127,6 +129,32 @@ If at any point in the future you want to be removed form the whitelist, you can
+
+Users can then keep these GRC, sell them via an exchange, donate them, or use them to purchase goods or services from vendors which accept GRC.
+
+The formula for figuring out the daily rewards your users will receive in total is:
+
+`(current GRC value) * (28750/ number of whitelisted projects)`
+
+You can find GRC's current and historical price through an exchange monitoring site like <a href="https://coinmarketcap.com/currencies/gridcoin/" rel="nofollow">CoinMarketCap</a>. Gridcoin does not control the price of GRC, it is determined by market conditions.
+
+## Can I increase the amount of rewards my user's receive?
+
+Absolutely! Increasing the reward amount can incentivize additional crunching on your project and is often much cheaper than purchasing additional computation from cloud and compute providers, as users are often crunching at a loss or break-even. You can do this via the `rainbymagnitude` feature, which allows you to distribute additional rewards to users based on their relative contribution to your project. This will work just as if the amount of GRC allocated to your project the day of your "rain" is +x where x is the amount you are "raining". Note that this does not provide _guaranteed_ compute, but does help incentivize people to compute more.
+
+1. Choose an amount to add to your total incentive. We suggest at least the current total amount of your incentive. So if your project normally receive x GRC per month, make your "rain" x GRC as well.
+2. Let your users know that you will be dispensing additional rewards, the amount of the additional rewards, and when the amount will be "rained". You can do this via an announcement on your site. It's probably a good idea to also post about it on Gridcoin's reddit, discord, and other communities. Giving users a heads up of several weeks will give them time to reconfigure their machines and increase their recent average credit.
+3. Purchase GRC from an <a href="../guides/acquire-grc.htm">exchange</a> or a community member. If you buy them on an exchange, you'll need to move them to your wallet.
+4. On the day you wish to dispense the rewards, open your Gridcoin wallet and wait until it is fully synced (cloud icon bottom right of wallet).
+5. In the wallet, go to Hamburger menu (top left) -> Help -> Debug Window -> Console
+6. Type in the command `listprojects` to find your project name
+7. Type in the command `rainbymagnitude "your_project_name" amount "message"`. For example: `rainbymagnitude "universe@home" 300 "Thank you for crunching universe!"`
```

Comment:
We have not supported messages in rains since 5.3.2

Reply:
Fixed